cmake/developer_package/packaging/rpm_post_build.cmake

```diff
@@ -37,6 +32,5 @@ foreach(rpm_file IN LISTS CPACK_PACKAGE_FILES)
 endforeach()
 
 if(NOT rpmlint_passed)
-    # till there is no filtering/suppression for rpmlint errors - throw warning instead fatal error
-    message(WARNING "rpmlint has found some mistakes. You can get more info regarding issues on site https://fedoraproject.org/wiki/Common_Rpmlint_issues")
+    message(FATAL_ERROR "rpmlint has found some mistakes. You can get more info regarding issues on site https://fedoraproject.org/wiki/Common_Rpmlint_issues")
```

Comment:
Please be sure that if works with merged into CI generating RPM/DEB

Reply:
will be shown on CI